Sources/App/Views/Home/HomeIndex+Model.swift

```diff
@@ -0,0 +1,41 @@
+import Fluent
+import Plot
+
+
+extension HomeIndex {
+    struct Model {
+        var recentPackages: [DatedLink]
+        var recentReleases: [DatedLink]
```

Comment:
Tempted to rename `DatedLink` to `DateLink` - or am I being too obsessed with naming? 😅

Reply:
DatedLink is better 👍